query-tool-results/participant-level-results.tsv

```diff
@@ -0,0 +1,11 @@
+DatasetID	SubjectID	Age	Sex	Diagnosis	Assessment	SessionID	SessionPath	NumSessions	Modality
+http://neurobagel.org/vocab/81e47a09-658d-48ea-b7c5-8d54820d038e	sub-01	3.41E1	http://purl.bioontology.org/ontology/SNOMEDCT/248152002		https://www.cognitiveatlas.org/task/id/4321, https://www.cognitiveatlas.org/task/id/1234	ses-01	/data/neurobagel/bagel-cli/bids-examples/synthetic/sub-01/ses-01	2	http://purl.org/nidash/nidm#FlowWeighted, http://purl.org/nidash/nidm#T1Weighted
```

Comment:
> `3.41E1`

Is this what we get back from the graph? Looks a bit odd in an example for human consumption

Reply:
Yeah, actually this is what the result ends up looking like from the API for a graph DB endpoint (this is what I was using to test our mini deployment). Not sure if it'd be worth manually editing the values here to in standard notation.